praas-app-ionic/src/store/conduit/edit.js

```diff
@@ -17,19 +17,15 @@ export const updateConduitFailure = (error) => ({
   payload: error,
 });
 
-export const updateConduit = (conduit, actions, changeMode) => {
+export const updateConduit = (conduit) => {
   return (dispatch) => {
     dispatch({ type: UPDATE_CONDUIT_REQUEST, payload: conduit });
```

Comment:
I find this code here which passes the conduit and spreads it in the reducer. But the initial state here doesnt adhere to this. Can we either update the initial state or clean this up unless its necessary?